src/v3/jest.setup.js

```diff
@@ -30,3 +31,13 @@ global.COMMITHASH = 'b9bbc0140703c3fbf0e2e58920362e70'; // "echo jest | md5"
 global.DEBUG = false;
 
 expect.addSnapshotSerializer(createSerializer({ includeStyles: false }));
+
+jest.mock('okta', () => {
+  return {
+    ...jest.requireActual('okta'),
+    loc: jest.fn().mockImplementation(
+      // eslint-disable-next-line no-unused-vars
+      (key, bundle, params) => (mockBundles.login[key] ? key : new Error(`Invalid i18n key: ${key}`)),
+    ),
+  };
+});
```

Comment:
This mock moved out of `__mocks__/okta.js` to be inlined here in the setup file. The other mock strategy was affecting v2 tests and mocking the `loc` function there as well.